rabbit-escape-engine/src/rabbitescape/engine/solution/SolutionInterpreter.java

```diff
@@ -45,9 +46,20 @@ public SolutionInterpreter( Solution solution, boolean appendWon )
         this.command = null;
         this.untilCount = -1;
     }
+    
+    public static SolutionInterpreter getNothingPlaying()
+    {
+        SolutionInterpreter nothingPlaying = new SolutionInterpreter( SolutionParser.parse( "1" ) );
+        nothingPlaying.emptySteps = true;
+        return nothingPlaying;
+    }
```

Comment:
Ah I thought emptySteps meant "send empty steps if you've run out" but it sounds like it means "send empty steps from now on" so my comments are irrelevant.